new code

```diff
@@ -0,0 +1,163 @@
+import java.util.*;
+public class tictactoenew
+{
+    public static void main(String args[])
+    { 
+        String rerun = "yes";
```

Comment:
this can be boolean
its like true or false
boolean reRun = true;
(notice not rerun its reRun)